README.md

```diff
@@ -57,16 +57,14 @@ To check that the warning is correct for SAVE - your static analyzer must print
 
 // FixMe: specify options here
 For [save-cloud](https://github.com/cqfn/save-cloud) it is extremely important to detect properties file for SAVE,
-so in case you would like to use both cli application and cloud application we suggest putting the file with name `save.properties` in the root folder of your repository.
-
-Configuration file can be also provided to SAVE via command line: `save -prop $PATH_TO_PROPS/save.properties`
+so in case you would like to use both cli application and cloud application we suggest putting the file with name `save.properties`
```

Comment:
We `suggest` or `demand`?

Reply:
I mean, we try to ask politely :) But actually, `save.properties` isn't that needed for cloud anymore, because we only needed this to discover root config. Now we can just pass path to test root dir and it will be enough for cloud. I'll change this sentence in readme

Reply:
Ok, just thought, that we need some place in README, where we say to user, that for proper work, if `save.properties` is provided, it should be at the same dir of root toml config

Reply:
Yes, I've added a line about it